app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java

```diff
@@ -83,7 +96,11 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         final ArrayList<File> pdfFiles = new ArrayList<>();
         final File[] files = folder.listFiles();
         if (files.length == 0) {
-            emptyStatusTextView.setVisibility(View.VISIBLE);
+
+            backView.setVisibility(View.VISIBLE);
```

Comment:
Instead of typing these 4 lines again, call the `setEmptyStateVisible` function.